### PR TITLE
[flink-doc] Adjust the document with markdown html table and table op…

### PR DESCRIPTION
**Pull request by YesOrNo828**
Updated Content:
1. Using the simple table format in the markdown replace the HTML table.
2. Update directory structure.
3. Update the arctic table options and notes.

Issue: #88

### CODE REVIEW

File: site/docs/ch/flink.md [hunk 1]
```diff
@@ -113,29 +115,33 @@ SHOW DATABASES;
 ```sql
 SHOW TABLES;
 ```
-## <a name="log">LOG 实时数据</a>
-**todo**：添加链接，format
+## LOG 实时数据<a name="log"></a>
```
Comment (zstraw) [inline site/docs/ch/flink.md:119]: I prefer adding a link to the architecture introduction of Arctic here to make log-store more clear to users.

File: site/docs/ch/flink.md [hunk 2]
```diff
@@ -113,29 +115,33 @@ SHOW DATABASES;
...
-实时数据可以提供毫秒级的数据可见性，并能保证数据一致性。可在不开启 Kafka 事务的情况下，保证数据的一致性。
-
-其底层存储可以对接外部消息队列中间件，当前仅支持 Kafka。用户可以通过在创建 Arctic 表时[配置](#CREATE TABLE)下述参数开启 Log：
+实时数据可以提供毫秒级的数据可见性，可在不开启 Kafka 事务的情况下，保证数据的一致性。
+
+其底层存储可以对接外部消息队列中间件，当前仅支持 Kafka。用户可以通过在[创建](#CREATE TABLE) Arctic 表时配置下述参数开启 LogStore：
+
+|Key|默认值|类型|是否必填|描述|
+|--- |--- |--- |--- |--- |
+|log-store.enable|false|Boolean|否|是否打开 Log Store，目前支持只Kafka|
```
Comment (zstraw) [inline site/docs/ch/flink.md:127]: typo:"目前支持只Kafka"

File: site/docs/ch/flink.md [hunk 3]
```diff
@@ -593,12 +432,6 @@ stream.print();
 // 提交并执行任务
 env.execute("Test Arctic Stream Read");
 ```
-properties 支持的参数，**当前只对非主键表生效**:
```
Comment (zstraw) [inline site/docs/ch/flink.md:596]: Shall we replace it with anchor before?

File: site/docs/ch/flink.md [hunk 4]
```diff
@@ -657,21 +490,17 @@ FlinkSink
 
 env.execute("Test Arctic Append");
 ```
-
-其中 table.properties 中支持写入的参数有：
```
Comment (zstraw) [inline site/docs/ch/flink.md:661]: Shall we replace it with anchor before?

**Review by zstraw — APPROVED** [review]

**Review by zhoujinsong — APPROVED** [review]

**Pull request merged**